dstep/translator/MacroDefinition.d

```diff
@@ -2053,6 +2053,10 @@ bool translateFunctAlias(
     {
         Identifier ident = cast(Identifier) expr.expr;
 
+        import dstep.core.Traits : isKeyword;
+        if (ident.spelling.isKeyword())
+            return false;
```

Comment:
But with this change, it's not possible to handle when it _is_ defined. Take this example from test suite, slightly modified:

```c
#define delegate(major, minor)#major"."#minor
#define STRINGIZE(major, minor)delegate(major, minor)
```

Currently this is translated to this:

```d
extern (D) string delegate(T0, T1)(auto ref T0 major, auto ref T1 minor)
{
    import std.conv : to;

    return to!string(major) ~ "." ~ to!string(minor);
}

alias STRINGIZE = delegate;
```

Which is, of course, a bug. Instead it should be translated with a `_` suffix for identifier `delegate`:

```d
extern (D) string delegate_(T0, T1)(auto ref T0 major, auto ref T1 minor)
{
    import std.conv : to;

    return to!string(major) ~ "." ~ to!string(minor);
}

alias STRINGIZE = delegate_;
```